.github/workflows/csv-to-csvw.yml

```diff
@@ -52,52 +53,68 @@ jobs:
             file_name="${file_path##*/}"
             file_extension="${file##*.}"
             out_path="out/${file_path}/"
-            
+            root_path=$(echo "$file_path" | cut -d "/" -f1)
```

Comment:
I'd be tempted to call this `top_folder` or similar instead of `root_path`. It may also be worth providing an example of the `file_path` variable in a comment so it's easier to understand what this is doing.